docusaurus.config.js

```diff
@@ -55,14 +55,7 @@ module.exports = {
         autoCollapseCategories: true,
       },
     },
-    announcementBar: {
```

Comment:
Should we comment out this section so that we can easily resurrect it for a future announcement?

Reply:
Yes, I agree - et us comment out rather than remove.

Also - @djmagee are we not converting this to a message about Cloud now?

Reply:
Yes, I agree - let us comment out rather than remove.

Also - @djmagee are we not converting this to a message about Cloud now?